### ISSUE TITLE
feature request: ConfigService

### ISSUE THREAD

**Issue by kbalk** [localstack/localstack#4409]
`moto` provides support for several of the AWS ConfigService APIs.  Would it be possible to provide that same support with LocalStack?

**Comment by kbalk** [comment]
Any update?

**Comment by kornicameister** [comment]
Not sure if that's related but right now it seems to be impossible to deploy a cloudformation that uses `AWS::Config::ConfigRule`.

```
localstack  | 2021-08-19T20:20:37:WARNING:localstack.utils.cloudformation.template_deployer: Unexpected resource type Config::ConfigRule when resolving references of resource
```

**Comment by kbalk** [comment]
I'm working on adding some moto ConfigService APIs related to ConfigRule - `put_config_rule`, `describe_config_rule` and `delete_config_rule` and should probably be done with that work by the weekend.  I don't know if that would help.

**Comment by kbalk** [comment]
The moto PR for ConfgRule was merged and will be released with moto 2.2.4.  

The localstack Cloudformation code would need to be modified to use it.  I need ConfigRule logic in localstack because we use both localstack and moto and if you use custom config rules, then it expects to find the custom config rule in localstack, not moto.

**Comment by kornicameister** [comment]
Ditto here, I'd at least love to se an ability to work with ConfigRules
